include/swift/AST/Attr.def

```diff
@@ -376,6 +376,10 @@ SIMPLE_DECL_ATTR(_forbidSerializingReference, ForbidSerializingReference,
   OnAnyDecl |
   LongAttribute | RejectByParser | UserInaccessible | NotSerialized,
   77)
+SIMPLE_DECL_ATTR(_hasInitialValue, HasInitialValue,
+  OnVar |
+  SILOnly | UserInaccessible,
```

Comment:
Oops, I think `UserInaccessible` is fine here. No need to limit it to SIL mode; no one's actually going to try to type it.

Reply:
👍